tests/includes/wait-for.sh

```diff
@@ -273,7 +276,7 @@ wait_for_systemd_service_files_to_appear() {
 		attempt=$((attempt + 1))
 	done
 
-	# shellcheck disable=SC2046
+	# shellcheck disable=SC2046,2005
```

Comment:
What is the 2005 for, and wouldn't it need to be `SC2005`?

Reply:
Either seems to work